.jscsrc

```diff
@@ -0,0 +1,74 @@
+{
```

Comment:
You have to set them to `null`. Hopefully the error messages one day will make that more obvious...